### PR TITLE
Added Cart Management

### PR DESCRIPTION
**Pull request by atharvabhanage02**
Added 
- Cart management 
- Order Details  

Here is the Preview Link : 
https://deploy-preview-4--distracted-wilson-c25ae0.netlify.app/cart/cart.html

### CODE REVIEW

File: cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    font-weight: bolder;
+    border-top: 1px solid var(---grey-border);
+    border-bottom: 1px solid var(---grey-border);
+}
+h2{
+    text-align: center;
+    margin: 2.5rem;
+}
+h3{
+    font-weight: bolder;
```
Comment (dev-enforced) [inline cart/cart.css]: Use a numeric value of font-weight instead of bolder word and store that value as a variable and use it!!

**Review by dev-enforced — APPROVED** [review]
The code looks good to me !! Good work Atharva on developing the cart screen!!
Had a small query mentioning below

File: cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+}
+.order-details>h3,label{
+    text-align: center;
+}
+.total-amount{
+    font-weight: bolder;
+    border-top: 1px solid var(---grey-border);
+    border-bottom: 1px solid var(---grey-border);
+}
+h2{
```
Comment (dev-enforced) [inline cart/cart.css:57]: So this property will be applied to all the h2 tags within this screen or some other screens as well? if no then it's okay to keep it?If yes then you can use this as a utility class and pass it to all the h2 tags within that screen !!

**Review by shaksham08 — APPROVED** [review]
looks good to me just follow the comments given above by dev-enforced

File: cart/cart.css (hunk 1 above)
Comment (atharvabhanage02) [reply cart/cart.css]: Yeah , I'll do that

**Pull request merged**